java/ql/src/semmle/code/java/security/ResponseSplitting.qll

```diff
@@ -0,0 +1,11 @@
+import semmle.code.java.dataflow.DataFlow
+
+/**
+ * Header-splitting sinks. Expressions that end up in an HTTP header.
```

Comment:
This qldoc does not adhere to our style guide. https://github.com/github/codeql/blob/master/docs/qldoc-style-guide.md